frontend/test/metabase/scenarios/admin/settings/sso/google.cy.spec.js

```diff
@@ -12,29 +12,36 @@ describe("scenarios > admin > settings > SSO > Google", () => {
       "fake-client-id.apps.googleusercontent.com",
     );
     saveSettings();
-
-    // This string lingers for far too long in the UI, so we have to wait for it to disappear before we assert on that same button again.
-    // Otherwise, the test fails. That's why we added a custom timeout of 6s.
-    cy.findByText("Success", { timeout: 6000 }).should("not.exist");
-
+    cy.findByText("Success");
+    cy.reload();
     cy.findByDisplayValue("fake-client-id.apps.googleusercontent.com").type(
       "fake-client-id2.apps.googleusercontent.com",
     );
     saveSettings();
+    cy.findByText("Success");
   });
 
-  it(
-    "Google sign-in client ID form should show an error message if it does not end with the correct suffix (metabase#15975)",
-  ),
-    () => {
-      cy.findByLabelText("Client ID").type("fake-client-id");
-      saveSettings();
+  it("Remove Google Sing-In Setup (metabase#20442)", () => {
+    cy.findByLabelText("Client ID").type("example.apps.googleusercontent.com");
+    cy.findByLabelText("Domain").type("example.test");
+    saveSettings();
+    cy.findByText("Success");
+    cy.reload();
```

Comment:
This part was already covered in the previous test.
There is no need to use UI to test the same flow multiple times. This was a good opportunity to use API.
```js
cy.request("PUT", "/api/setting", {
  "google-auth-client-id": "example.apps.googleusercontent.com",
  "google-auth-auto-create-accounts-domain": "example.test",
});
```